devsiteHelper.py

```diff
@@ -231,9 +231,13 @@ def renderDevSiteContent(content, lang='en'):
     fbContent = re.search(r'({%[ ]?framebox.+?%})(.*?){%[ ]?endframebox[ ]?%}(?ms)', framebox)
     fbOpenTag = fbContent.group(1)
     fbHeight = re.search(r'height="(.*?)"', fbContent.group(1))
+    fbClass = re.search(r'class="(.*?)"', fbContent.group(1))
     fbContent = fbContent.group(2)
-    fbMemcacheKey = '/framebox/' + hashlib.md5(fbContent.encode('utf-8')).hexdigest()
```

Comment:
Uhm no, I definitely wasn’t intending to remove that. Must have resolved the conflict the wrong way around while rebasing -.- Thanks Pete!